jsonpath/0.8.1.wso2v1/pom.xml

```diff
@@ -0,0 +1,120 @@
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied.  See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.jayway.jsonpath</groupId>
+    <artifactId>com.jayway.jsonpath</artifactId>
+    <packaging>bundle</packaging>
+    <name>json-path.wso2</name>
+    <version>0.8.1.wso2v1</version>
+    <description>
+        This bundle will represent json-path
+    </description>
+    <url>http://wso2.org</url>
+
+  <dependencies>
+	<dependency>
+    		<groupId>com.jayway.jsonpath</groupId>
+    		<artifactId>json-path</artifactId>
+    		<version>0.8.1</version>
+	</dependency>
+	<dependency>
+    		<groupId>com.jayway.jsonpath</groupId>
+    		<artifactId>json-path-assert</artifactId>
+    		<version>0.8.1</version>
+    		<scope>test</scope>
+		<optional>true</optional>
+	</dependency>
+	
+
+	<dependency>
+		<groupId>commons-lang</groupId>
+		<artifactId>commons-lang</artifactId>
+		<version>2.6</version>
+		<optional>true</optional>
+	</dependency>
+            
+            
+ </dependencies>
+	
+   <repositories>
+        <repository>
+            <id>wso2-nexus</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>fail</checksumPolicy>
+            </releases>
+        </repository>
+    </repositories>
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+        <snapshotRepository>
+            <id>wso2.snapshots</id>
+            <name>Apache Snapshot Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+        </snapshotRepository>
+    </distributionManagement>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+	        <version>1.4.0</version>               
+                <extensions>true</extensions>
+		
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
```

Comment:
Better to organize the imports one by one so it will be easy for others to check packages.